app.js

```diff
@@ -116,8 +116,8 @@ if (Configuration.get('FEATURE_MORGAN_LOG_ENABLED')) {
 	}));
 }
 
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json({ limit: '10mb' }));
+app.use(bodyParser.urlencoded({ extended: true, limit: '10mb' }));
```

Comment:
this is also checked in the backend? Does the file upload still work in other places, such as the contact form?